conf/config-default.yaml

```diff
@@ -174,6 +174,8 @@ nginx_config:                     # config for render the template to generate n
     # lua_shared_dicts:            # add custom shared cache to nginx.conf
     #  ipc_shared_dict: 100m       # custom shared cache, format: `cache-key: cache-size`
 
+    proxy_ssl_server_name: true   # disable passing of the server name through tls
```

Comment:
missing test cases

Reply:
Testing this feature means we test the behavior of the tls which returned ca. I havn't found a good way to deal with it, because I thought it was hard to deep into tls.